minai_plugin/customintegrations.php

```diff
@@ -96,6 +98,11 @@ function ProcessIntegrations() {
         );
         $MUST_DIE=true;
     }
+    if (isset($GLOBALS["gameRequest"]) && $GLOBALS["gameRequest"][0] == "updateThreadsDB") {
```

Comment:
I haven't played around with the command@ syntax yet! Will this be "updateThreadsDB" with how we're calling from papyrus?

Reply:
yep, I already wrote function for that in papyrus here where we call it https://github.com/MinLL/MinAI/pull/49/files/a68a2c852d93733ec8c71189f0c6c74bef75584f#diff-b5b3638a1cc05542ef052d9ffda4fb47205eedcfff4c954e21b024be90a1374cR1169

and that function is called when events happen in scene(scene change, speed change, start, end)